android/libraries/rib-android/src/main/kotlin/com/uber/rib/core/lifecycle/ActivityLifecycleEvent.kt

```diff
@@ -79,7 +109,7 @@ private constructor(
         Type.DESTROY -> DESTROY_EVENT
         else ->
           throw IllegalArgumentException(
-            "Use the createOn${type.name.toLowerCase().capitalize()}Event() method for this type!",
+            "Use the createOn${type.name.lowercase().replaceFirstChar(Char::titlecase)}Event() method for this type!",
```

Comment:
Kind of surprised they replaced capitalize with this, it's much more verbose and less obvious for folks.

Reply:
Actually the drop in replacement is even more verbose (suggested by ReplaceWith).

From googling It, it seems the original deprecated capitalize function had issues with
1. Not taking a locale 
2. Surprising behavior on characters that are one char in Unicode, but actually two chars in the natural language.

Both of which don't matter at this particular call site of ours